### PR TITLE
Add pre noted conversion test

### PR DESCRIPTION
**Pull request by NohYujeong**
- bubble blocker
- sudden alert
- bottom banner
- remove wing banner at paper show

### CODE REVIEW

File: app/components/paperShow/refCitedTab/index.tsx [hunk 1]
```diff
@@ -24,37 +26,57 @@ const TabItem: React.FunctionComponent<TabItemProps> = props => {
...
     return (
-      <div className={styles.actionItem}>
-        <PdfDownloadButton
-          paper={props.paper}
-          isLoading={props.isLoading}
-          onDownloadedPDF={props.onClickDownloadPDF!}
-          handleSetScrollAfterDownload={props.afterDownloadPDF}
-        />
-      </div>
+      <ClickAwayListener onClickAway={() => setIsOpenBlockedPopper(false)}>
```
Comment (TylorShin) [inline app/components/paperShow/refCitedTab/index.tsx]: same here

File: app/components/paperShow/refCitedTab/index.tsx [hunk 2]
```diff
@@ -24,37 +26,57 @@ const TabItem: React.FunctionComponent<TabItemProps> = props => {
...
+            buttonClickAction={"downloadPdf"}
+          />
+        </div>
+      </ClickAwayListener>
     );
   }
   return (
-    <>
-      <div className={styles.actionItem}>
+    <ClickAwayListener onClickAway={() => setIsOpenBlockedPopper(false)}>
```
Comment (TylorShin) [inline app/components/paperShow/refCitedTab/index.tsx]: I think it's too duplicated code. the only different thing here is the content of the button.

File: app/components/pdfViewer/index.tsx [hunk 3]
```diff
@@ -286,34 +293,72 @@ const PDFViewer: React.FunctionComponent<PDFViewerProps> = props => {
...
                 ) : (
-                  <ScinapseButton
-                    gaCategory="PDF viewer"
-                    gaAction={actionTag}
-                    style={readAllBtnStyle}
-                    content={
-                      <span>
-                        READ ALL <Icon icon="ARROW_POINT_TO_UP" className={styles.arrowIcon} />
-                      </span>
+                  <ClickAwayListener
```
Comment (TylorShin) [inline app/components/pdfViewer/index.tsx]: same here

File: app/components/pdfViewer/index.tsx [hunk 4]
```diff
@@ -286,34 +293,72 @@ const PDFViewer: React.FunctionComponent<PDFViewerProps> = props => {
...
+                        gaAction={actionTag}
+                        style={readAllBtnStyle}
+                        content={
+                          <span>
+                            READ ALL <Icon icon="ARROW_POINT_TO_UP" className={styles.arrowIcon} />
+                          </span>
+                        }
+                        isLoading={PDFViewerState.isLoading}
+                        disabled={PDFViewerState.hasFailed}
+                        onClick={async () => {
```
Comment (TylorShin) [inline app/components/pdfViewer/index.tsx]: pull out this logic to other function.
this pattern makes the content be difficult to read.

File: app/components/preNoted/blockedPopper.tsx [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+
+  return (
+    <div className={styles.bubbleContent}>
+      <div className={styles.titleContext}>{bubbleContext.title}</div>
+      <div className={styles.mainContext}>{bubbleContext.mainText}</div>
+      <button
+        className={styles.joinBtn}
+        onClick={e => {
+          e.preventDefault();
+          blockUnverifiedUser({
```
Comment (TylorShin) [inline app/components/preNoted/blockedPopper.tsx]: missing async await logic.

Comment (TylorShin) [reply app/components/preNoted/blockedPopper.tsx]: is it right the QA is passed?
before asking QA or PR please test all features by yourself.

i guess this function cannot block unverified user.

File: app/helpers/getBubbleContextType.tsx [hunk 6]
```diff
@@ -0,0 +1,13 @@
+export const BUBBLE_CONTEXT_TYPE = "b_c_t";
+
+const store = require("store");
+
+export function setBubbleContextTypeHelper() {
+  let bubbleContextType: number = store.get(BUBBLE_CONTEXT_TYPE) || 1;
```
Comment (TylorShin) [inline app/helpers/getBubbleContextType.tsx]: people who read this code cannot understand what 1, 2, 3 means.
use `enum` or at least `CONSTANT` for a comprehensible code.

File: app/containers/paperShowCollectionControlButton/index.tsx [hunk 7]
```diff
@@ -73,27 +79,46 @@ const TitleArea: React.SFC<TitleAreaProps> = props => {
...
-        }}
-        className={styles.unsignedTitleBtn}
-      >
-        <Icon icon="COLLECITON_LIST" className={styles.collectionIcon} />
-        Add to Collection
-      </button>
+      <ClickAwayListener onClickAway={() => setIsOpenBlockedPopper(false)}>
+        <div ref={addToCollectionBtnEl}>
+          <button
+            onClick={() => {
```
Comment (TylorShin) [inline app/containers/paperShowCollectionControlButton/index.tsx]: fat code.
and also doesn't wait `blockUnverifiedUser`

File: app/containers/paperShowCollectionControlButton/index.tsx [hunk 8]
```diff
@@ -73,27 +79,46 @@ const TitleArea: React.SFC<TitleAreaProps> = props => {
...
-            actionTag: "addToCollection",
-            actionLabel: null,
-          });
-        }}
-        className={styles.unsignedTitleBtn}
-      >
-        <Icon icon="COLLECITON_LIST" className={styles.collectionIcon} />
-        Add to Collection
-      </button>
+      <ClickAwayListener onClickAway={() => setIsOpenBlockedPopper(false)}>
```
Comment (TylorShin) [inline app/containers/paperShowCollectionControlButton/index.tsx]: same here

File: app/components/preNoted/suddenAlert.tsx [hunk 9]
```diff
@@ -0,0 +1,55 @@
...
+import { useIntervalProgress } from "../../hooks/useIntervalProgressHook";
+const styles = require("./suddenAlert.scss");
+
+interface SuddenAlertProps extends PopperProps {}
+
+const SuddenAlert: React.FC<SuddenAlertProps> = props => {
+  const [renderTime, setRenderTime] = React.useState(0);
+  const [shouldShowSuddenAlert, setShouldShowSuddenAlert] = React.useState(false);
+
+  useIntervalProgress(() => {
```
Comment (TylorShin) [inline app/components/preNoted/suddenAlert.tsx]: you don't need to use `setInterval` for this.
why don't you just use `setTimeout` instead?

File: app/components/preNoted/blockedPopper.tsx (hunk 5 above)
Comment (NohYujeong) [reply app/components/preNoted/blockedPopper.tsx]: I have found that it works very well when I tested the relevant logic.

However, it is a good idea to write the code clearly, so I will add that logic.

File: app/components/preNoted/suddenAlert.tsx (hunk 9 above)
Comment (NohYujeong) [reply app/components/preNoted/suddenAlert.tsx]: Thanks for a good suggestion.

**Pull request merged**